tools/nl/nodejs_query_differ/goldens/bard/timeline_all.json

```diff
@@ -97,9 +97,9 @@
       "chartUrl": ""
     },
     {
-      "data_csv": "label,Median Married Couple Family Household Income\r\n2019,96825\r\n2018,93643\r\n2017,89787\r\n2016,86919\r\n2015,84437\r\n2014,82236\r\n2013,79820\r\n2012,76946\r\n2011,73585\r\n2010,69952",
+      "data_csv": "label,Average Income for Households\r\n2019,85476\r\n2018,84043\r\n2017,81334\r\n2016,78828\r\n2015,76185\r\n2014,73272\r\n2013,70235\r\n2012,67426\r\n2011,64106\r\n2010,60581",
```

Comment:
to confirm, are these diff expected?

Reply:
this makes sense because the diff makes the bard golden the same as the autopush golden